spec/javascripts/Modules.AllocationDataTable_spec.js

```diff
@@ -82,7 +82,19 @@ describe("Modules.AllocationDataTable.js", function () {
         }
       };
       var output = options.createdRow(row, data);
-      expect(output[0].outerHTML).toEqual('<tr class="cav_warning"><td><div class="warning-message-container"><div class="warning-message">CAVs not injected</div></div></td></tr>');
+      expect(output[0].outerHTML).toEqual('<tr class="injection_warning"><td><div class="warning-message-container"><div class="warning-message">CAVs not injected</div></div></td></tr>');
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3478/files#r477154369)